modules/ROOT/pages/manage-client-apps-latest-task.adoc

```diff
@@ -3,12 +3,20 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Exchange, you can make information about APIs you develop in Anypoint Platform publicly available for applications that want to consume the API. You can limit access to certain users.
+In Exchange, you can make information about APIs you develop in Anypoint Platform publicly available for applications that want to consume the API. You can limit access to certain users. +
+The applications, to which you have granted access, are listed on API Manager > *Contracts*
 
-The applications, to which you have granted access, are listed on API Manager > Contracts. As Organization Administrator of the master organization, you can manage contracts registered to access your API.
+===== Steps
```

Comment:
Remove this lvl 5 anchor. I would start these steps by adding a numbered list.
Something following the style of this excerpt: 
> To manage your API Instance contracts: 
>
> . In API Manager, click **API Administration**.
> . Click the API Name of your desired API instance.
> . Click the API Version of your selected API instance. +
> This opens the configuration page of your selected API instance.
> . Click **Contracts** from the left menu. +
> This tab displays information about every contract with your API instance, and a client application.